openstack/imageservice/v2/images/requests.go

```diff
@@ -0,0 +1,301 @@
+package images
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/rackspace/gophercloud"
+	"github.com/rackspace/gophercloud/pagination"
+)
+
+// ListOptsBuilder allows extensions to add additional parameters to the
+// List request.
+type ListOptsBuilder interface {
+	ToImageListQuery() (string, error)
+}
+
+// ListOpts allows the filtering and sorting of paginated collections through
+// the API. Filtering is achieved by passing in struct field values that map to
+// the server attributes you want to see returned. Marker and Limit are used
+// for pagination.
+//http://developer.openstack.org/api-ref-image-v2.html
+type ListOpts struct {
```

Comment:
`Visibility`, `Status`, and `MemberStatus` are all restricted to a handful of values. Maybe we should make types for those fields.